apis/projectcontour/v1alpha1/accesslog.go

```diff
@@ -251,8 +251,10 @@ func (a AccessLogLevel) Validate() error {
 const (
 	// Log all requests. This is the default.
 	LogLevelInfo AccessLogLevel = "info"
-	// Log only requests that result in an error.
+	// Log only requests that result in an non-success response code.
```

Comment:
```suggestion
	// Log only requests that result in a non-success (i.e. 300+) response code.
```

Reply:
done in db1d291e83351f7a0da9db8559b4ca6a455a4fec